coordinator/src/message.rs

```diff
@@ -3,19 +3,19 @@ use bitcoin::secp256k1::PublicKey;
 use futures::future::RemoteHandle;
 use futures::FutureExt;
 use orderbook_commons::Message;
+use parking_lot::RwLock;
 use std::collections::HashMap;
 use std::sync::Arc;
-use std::sync::RwLock;
 use tokio::sync::broadcast;
 use tokio::sync::mpsc;
 
-/// This value is arbitrarily set to 100 and defines the message accepted in the notification
+/// This value is arbitrarily set to 100 and defines the message accepted in the message
 /// channel buffer.
 const NOTIFICATION_BUFFER_SIZE: usize = 100;
 
-// TODO(holzeis): This enum should be extended to allow for sending push notifications.
-pub enum Notification {
-    Message {
+/// Message sent to users via the websocket.
+pub enum CoordinatorMessage {
```

Comment:
🔧 `CoordinatorMessage` is not correct here as this also includes messages from the orderbook. Actually the majority of messages are coming from the orderbook.

Reply:
I'll change it to `WebsocketMessage`, thank you fro the suggestion.

I'm not sure whether it would make sense to couple them together (in my mind push notifications are completely separate thing, that user can opt-in into; and also needs FCM key to work locally).

Reply:
edit: there was already `WebsocketMessage` somewhere else, so I ended up with `OrderbookMessage`.